implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Ockam.Services.API.NodeInfo.Info do
+  @moduledoc """
+  Data structure to communicate node information
+  Currently only contains version
+  """
+  use TypedStruct
+
+  typedstruct do
+    plugin(Ockam.TypedCBOR.Plugin)
+    field(:version, String.t(), minicbor: [key: 1])
+  end
+end
+
+defmodule Ockam.Services.API.NodeInfo do
+  @moduledoc """
+  API worker to show node information
+  Just returns the information provided on worker setup
+
+  Options:
+  - info: Ockam.Services.API.NodeInfo.Info - information to return
+  """
+  use Ockam.Services.API
+
+  alias Ockam.API.Request
+  alias Ockam.Services.API.NodeInfo.Info
+
+  def node_info(worker, timeout \\ 5000) do
+    Ockam.Worker.call(worker, :get, timeout)
+  end
+
+  @impl Ockam.Worker
+  def setup(options, state) do
```

Comment:
Zero functionality difference and no change necessary, but stylistically I probably would've written these as separate function heads and pattern match there. Do we have an internal style guide for that kind of super subjective stuff?

Reply:
🤦🏻  I totally overlooked it was a Keyword, sorry!